.github/scripts/brew.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -eux
+
+. .github/scripts/env.sh
+
+if [ -e "$HOME/.brew" ] ; then
```

Comment:
Here we re-use existing installation. So far this has not created any problems. If it does, this can be changed to `$CI_PROJECT_DIR/.brew`.

Reply:
All self-hosted runners are currently mutable.